framework/modules/mc/mc-hologram/src/main/java/io/fairyproject/mc/hologram/entity/impl/TextDisplayHologramEntity.java

```diff
@@ -101,10 +101,10 @@ public void hide(MCPlayer player) {
     private List<EntityData> createEntityData(MCPlayer mcPlayer) {
         List<EntityData> entityDataList = new ArrayList<>();
 
-        entityDataList.add(new EntityData(22, EntityDataTypes.COMPONENT, MCAdventure.asItemString(line.render(mcPlayer), mcPlayer.getLocale()))); // text
-        entityDataList.add(new EntityData(23, EntityDataTypes.CAT_VARIANT, 200)); // line width
-        entityDataList.add(new EntityData(24, EntityDataTypes.CAT_VARIANT, 0x40000000)); // background color
-        entityDataList.add(new EntityData(25, EntityDataTypes.CAT_VARIANT, -1)); // text opacity
+        entityDataList.add(new EntityData(23, EntityDataTypes.COMPONENT, MCAdventure.asItemString(line.render(mcPlayer), mcPlayer.getLocale()))); // text
```

Comment:
Is the metadata index the same in lower versions? 1.19 and 1.20 for example.

Reply:
you are right, the index has been changed in 1.20.2. I will update it soon

Reply:
should be good now :)

Reply:
You are the goodest